es6/context.js

```diff
@@ -64,7 +71,8 @@ export default {
       chalk.red(lineInfo.line.substr(lineInfo.column, length)) +
       lineInfo.line.substring(lineInfo.column + length, lineEnd);
     return {
-      info
+      info,
+      lineNumber: lineInfo.lineNumber
```

Comment:
I don't really like this, just surfacing one the pieces of lineInfo data, perhaps I should just expose lineInfo in its entirety? e.g.

```
return {
  info,
  lineInfo
}
```